src/tests/data/FactoryUtils.ts

```diff
@@ -0,0 +1,13 @@
+class FactoryUtils {
+  public static create<T>(n: number, fn: () => T): T[] {
+    return Array(n).fill(null).map(fn);
+  }
+
+  public static mapToValue(arr, val) {
+    return arr.map((x) => x[val]);
+  }
+  public static compareByDate(a, b) {
+    return -1 * (new Date(a.date).getTime() - new Date(b.date).getTime());
```

Comment:
why not just flip the subtraction? is there a reason why it's being done like this?

Reply:
no reason, i think i realized it was sorted the other way and added a -1